src/java/edu/ucsd/library/dams/api/DAMSAPIServlet.java

```diff
@@ -1524,7 +1531,15 @@ else if ( path.length == 3 && path[1].equals("objects") )
 				ts = triplestore(req);
 				es = events(req);
 				cacheRemove(path[2]);
+				boolean isCollection = isCollection(ts, es, path[2]);
+
 				info = objectDelete( path[2], ts, es, fs );
+				if ((int)info.get("statusCode") == SC_OK)
+				{
+					String[] ids = { path[2] };
+					String indexType = isCollection ? "purgeCollection" : "purgeObject";
```

Comment:
@lsitu , :+1:  To clarify as the delete parameter changed,  this line need to change to:  elsif method == "purgeObject" || "purgeCollection",  correct?
https://github.com/ucsdlib/damspas/blob/develop/bin/damsolrizer#L109